docs/architecture/decisions/0002-use-openapi-to-defined-the-api-specs.md

```diff
@@ -0,0 +1,19 @@
+# 2. Use OpenAPI to defined the API specs
```

Comment:
[This is the 1st](https://github.com/teamdigitale/cittadinanza-digitale/blob/master/docs/architecture/decisions/0001-record-architecture-decisions.md)